docs/service-mesh/03-01-istio-setup-in-kyma.md

```diff
@@ -14,19 +14,15 @@ This list shows the available Istio components and the components enabled by def
 
 | Component | Enabled |
 | :--- | :---: |
+| Istiod | ✅ |
 | Gateways | ✅ |
-| Sidecar Injector | ✅ |
-| Galley | ✅ |
-| Mixer | ✅ |
-| Pilot | ✅ |
-| Security | ✅ |
-| Node agent | ⛔️ |
 | Grafana | ⛔️ |
 | Prometheus | ⛔️ |
-| Servicegraph | ⛔️ |
 | Tracing | ⛔️ |
 | Kiali | ⛔️ |
 
+>*NOTE*: In Istio 1.5, separate components are replaced by a single binary - Istiod. However, to ensure a smooth transition to the new version, Citadel, Policy and Telemetry still are deployed to the cluster.
+
 ## Kyma-specific configuration
 
 These configuration changes are applied to customize Istio for use with Kyma:
```

Comment:
True, but it is set by default in Istio
```
Only the Ingress Gateway is enabled.
```

It is no longer a `Kyma-specific configuration`. It is by default and doesn't need any configuration at all:
```
- The [Secret Discovery Service](https://www.envoyproxy.io/docs/envoy/latest/configuration/security/secret.html#secret-discovery-service-sds) is enabled in the Ingress Gateway.
```

There are no exceptions in mtls config.
```
- [Mutual TLS (mTLS)](https://istio.io/docs/concepts/security/#mutual-tls-authentication) is enabled cluster-wide with the exception of the Istio Control Plane.
```

The resources are actually set to CPU: 200m, memory: 128Mi in limits and CPU: 20m, memory: 32Mi in requests
```
New resource limits for Istio sidecars are introduced: CPU: `100m`, memory: `128Mi`.
```

The DRs are created after the installation; DR for istio-ingressgateway is created only for minikube and DR for apiserver is created always
```
DestinationRules are created by default, which disables mTLS for the `istio-ingressgateway.istio-system.svc.cluster.local` service.
```

Reply:
There are also a few kyma-specific configurations not mentioned here.

- adjusted (decreased) resources for all Istio components 
- PILOT_HTTP10 flag set in Pilot (istiod) component environment variables
- `disablePolicyChecks` set to false (to enable old Policy checks)
- telemetry configured to use v1 version